apps/src/code-studio/components/ShareAllowedDialog.story.jsx

```diff
@@ -20,25 +17,14 @@ const fakei18n = {
     }
   };
 
-// Provide the redux store because the unconnected ShareAllowedDialog renders a
-// connected PublishDialog.
-
-function configureStore() {
-  return createStore(combineReducers({
-    publishDialog: publishDialogReducer,
```

Comment:
Right now it looks like you're providing a store without configuring any reducers.  How do you propose we let stories add the individual reducers they depend on?